.drone.yml

```diff
@@ -56,6 +58,7 @@ steps:
   commands:
   - export CARGO_TARGET_DIR=$PWD/drone/target
   - export PYTHON_VENV_DIR=$PWD/drone/venv
+  - export CRYPTO_CHAIN_ENABLE_SANITY_CHECKS=1
```

Comment:
drone hmac won't verify / needs updating -- I can possibly sign it later if you can't

Reply:
Still get this when trying to sign:
```
$ drone sign crypto-com/chain
client error 404: {"message":"Not Found"}
```